distributed/scheduler.py

```diff
@@ -6922,7 +6922,10 @@ def workers_to_close(
         if isinstance(key, bytes):
             key = pickle.loads(key)
 
-        groups = groupby(key, self.workers.values())
+        # Long running tasks are typically using a worker_client to
+        # scheduler other tasks. We should never downscale them
```

Comment:
```suggestion
        # Long running tasks typically use a worker_client to schedule
        # other tasks. We should never shut down the worker they're
        # running on, as it would cause them to restart from scratch
        # somewhere else.        
```